db/schema.rb

```diff
@@ -188,17 +188,17 @@
 
   create_table "question_group_conditionals", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8" do |t|
     t.integer  "rapidfire_question_group_id"
-    t.integer  "cohort_id"
+    t.integer  "campaign_id"
     t.datetime "created_at",                  null: false
     t.datetime "updated_at",                  null: false
-    t.index ["cohort_id"], name: "index_question_group_conditionals_on_cohort_id", using: :btree
+    t.index ["campaign_id"], name: "index_question_group_conditionals_on_campaign_id", using: :btree
```

Comment:
And the two others on campaigns_survey_assignments, I guess.

I'm pretty sure if running the migrations didn't change this, then the old indexes will still be there and the new ones won't be.